storybook/main.js

```diff
@@ -10,27 +10,34 @@ module.exports = {
       },
     },
     '@storybook/addon-a11y',
+    '@storybook/addon-webpack5-compiler-babel',
```

Comment:
this is because storybook uses Webpack 5 by default now?

Reply:
We were already using webpack 5 with storybook. When I ran the upgrade this was added automatically for me. More details here: https://storybook.js.org/docs/builders/webpack#babel

Reply:
okay thank you for the information.